tests/integration/backward_compatible/dbapi20.py

```diff
@@ -0,0 +1,831 @@
+#!/usr/bin/env python
```

Comment:
We need to add server and client version guards to this test (since it is in the backward compatible folder)

Reply:
Moved these tests out out of backward compatibility.